servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/StreamingHttpServiceAsyncContextTest.java

```diff
@@ -84,15 +87,24 @@ protected ServerContext serverWithEmptyAsyncContextService(HttpServerBuilder ser
     }
 
     private static StreamingHttpService newEmptyAsyncContextService() {
+        // The service should get an empty AsyncContext regardless of what is done outside the service.
+        // There are utilities that may be accessed in a static context or before service initialization that
+        // shouldn't pollute the service's AsyncContext.
+        AsyncContext.current().put(K1, "value");
```

Comment:
Consider testing that `<K1, "value">` is still present for the main thread after server started and the request complete.